pandas/core/frame.py

```diff
@@ -1242,7 +1242,7 @@ def from_dict(cls, data, orient="columns", dtype=None, columns=None):
 
         return cls(data, index=index, columns=columns, dtype=dtype)
 
-    def to_numpy(self, dtype=None, copy=False):
+    def to_numpy(self, dtype=None, copy=False, na_value=lib._no_default):
```

Comment:
Do we need to add it here? Currently it is not that useful (it is eg not using the EA.to_numpy ones, which we maybe should try to do in the future), so maybe only add it once useful?

Reply:
Could go either way here. I agree that it's not useful.

Reply:
Removed this for now.